docs/AQL/master03-syntax.adoc

```diff
@@ -451,14 +451,29 @@ AQL structure has the following clauses and these clauses must be listed in the
 
 === FROM
 
-The `FROM` clause is used to specify the domain of the query, that is a subset of the universe of data that could be retrieve from a CDR. That universe is anything defined in the openEHR RM (that is a set of classes that models clinical records), and archetypes (that define specific data sets based on the openEHR RM). Because of that, the `FROM` clause uses `class expressions` to specify the subsets of data, giving context to the query.
+The `FROM` clause is used to specify the subset of all the data available in a repository,
+that will be available for the rest of the clauses to filter and return specific data (`WHERE`, `SELECT`, etc).
+The data available should be defined by a Reference Model (RM), using the dual-model approach,
+that is constraining the RM by the use of archetypes to define specific data structures.
 
-Then, the expressions in the `WHERE` clause will filter data, but only from the subset defined in the `FROM`, to get only the data that we need. And in the `SELECT` clause, a final projection is applied, selecting only the matched data that should be retrieved. In addition to the filtering done in the `WHERE` clause, the `class expressions` could hold predicates that also help filtering data.
+All the classes referenced by the FROM clause, should be defined by the RM. For instance,
+if the Reference Model is the openEHR EHR Model, the FROM should only include classes like
+EHR, COMPOSITION, OBSERVATION, EVALUATION, and so on.
 
-Ass a summary:
+NOTE: The AQL specification is not binded to a specific Reference Model, but to use a
```

Comment:
'AQL specification is not bound' (another annoying, unpredictable EN-language verb ;)